shared/src/business/entities/cases/Case.js

```diff
@@ -823,9 +832,17 @@ Case.prototype.toRawObject = function (processPendingItems = true) {
 };
 
 Case.prototype.doesHavePendingItems = function () {
-  return this.docketEntries.some(
-    docketEntry => docketEntry.pending && docketEntry.servedAt,
-  );
+  if (isCodeEnabled(7198)) {
```

Comment:
These two blocks of code are so similar, it seems like it could be broken out into a method - thoughts?

Reply:
Do you mean a helper for `docketEntry.servedAt || docketEntry.isLegacyServed` ?

Reply:
Sure - if that makes sense to do.

Reply:
Refactoring card added https://trello.com/c/BQLzXAuD